Alerts table/page updates for 8.7 (#3058)
elastic/security-docs: docs/detections/alerts-visualizations.asciidoc

[role="screenshot"]
image::images/alert-page-visualizations.png[Alerts page with visualizations section highlighted]

Use the left buttons to select a view type (*Summary*, *Trend*, *Counts*, or *Treemap*), and use the right menus to select the ECS fields to use for grouping: 

* *Top alerts by* or *Group by*: Primary field for grouping alerts.
* *Group by top* (if available): Secondary field for further subdividing grouped alerts.

For example, you can group first by rule name (`Group by: kibana.alert.rule.name`), then by host name (`Group by top: host.name`) to visualize which detection rules generated alerts, and which hosts triggered each of those rules. For groupings with a lot of unique values, the top 1,000 results are displayed.

NOTE: Some view types don't have the *Group by top* option. You can also leave *Group by top* blank to group by only the primary field in *Group by*.

To reset a view to default settings, click the settings menu (image:images/three-dot-icon.png[Settings menu icon,18,18]) in the upper-right and select *Reset group by fields* (the settings menu appears when you hover over the area).

> === Summary

Reviewer: Do you need the [float] tag and an anchor here? Same question for the rest of the sections in this topic.

Reviewer 2: Good question! I didn't think they would all render on the same page without the [float] or [discrete] tags, so maybe I don't understand those tags as well as I thought

Author: Great catch @nastasha-solomon @benironside. I have no idea why they're rendering "correctly" without those tags, either -- another mystery of AsciiDoc. But I'm going to add tags to these sections as a best practice.

> The summary visualization is the default visualization on the Alerts page and shows how alerts are distributed across a few different indicators:

Reviewer: This is a great explanation of the visualizations.

Reviewer 2: Yeah this is 🚀 🎸 😎

* *Top alerts by* or *Group by*: Primary field for grouping alerts.
* *Group by top* (if available): Secondary field for further subdividing grouped alerts.

For example, you can group first by rule name (`Group by: kibana.alert.rule.name`), then by host name (`Group by top: host.name`) to visualize which detection rules generated alerts, and which hosts triggered each of those rules. For groupings with a lot of unique values, the top 1,000 results are displayed.

NOTE: Some view types don't have the *Group by top* option. You can also leave *Group by top* blank to group by only the primary field in *Group by*.

To reset a view to default settings, click the settings menu (image:images/three-dot-icon.png[Settings menu icon,18,18]) in the upper-right and select *Reset group by fields* (the settings menu appears when you hover over the area).

Click the collapse icon (image:images/collapse-icon-horiz-down.png[Collapse icon,16,15]) to minimize the visualization section and display a summary of key information instead.

[role="screenshot"]
image::images/alert-page-viz-collapsed.png[Alerts page with visualizations section collapsed]

[discrete]
=== Summary

> The summary visualization is the default visualization on the Alerts page and shows how alerts are distributed across a few different indicators:

Reviewer 3: ```suggestion
On the Alerts page, the summary visualization displays by default and shows how alerts are distributed across these indicators:
```
